basic_network.go

```diff
@@ -426,7 +433,7 @@ func handleSubReq(nw *BasicVideoNetwork, subReq SubReqMsg, remotePID peer.ID) er
 	peers, err := closestLocalPeers(nw.NetworkNode.PeerHost.Peerstore(), subReq.StrmID)
 	if err != nil {
 		glog.Errorf("Error getting closest local node: %v", err)
-		return err
+		return nil
```

Comment:
Is this intentional? Not returning the error?

Reply:
Yeah - "returning the error" == "dropping the incoming stream".  In this case, just because we didn't find any peers doesn't mean we should drop the incoming stream.

Reply:
Actually, I take it back.  We should return `ErrHandleMsg`.